pandas/io/clipboards.py

```diff
@@ -38,10 +38,8 @@ def read_clipboard(sep=r'\s+', **kwargs):  # pragma: no cover
 
     # Try to decode (if needed, as "text" might already be a string here).
     try:
-        text = compat.bytes_to_str(
-            text, encoding=(kwargs.get('encoding') or
-                            get_option('display.encoding'))
-        )
+        text = text.decode(kwargs.get('encoding') or
+                           get_option('display.encoding'))
     except AttributeError:
```

Comment:
May e take the getoption outside of the try/except?

Reply:
Hmm...that `encoding` logic is specific to the `try-except` block.  I'm not sure we should be putting it outside.

In addition, since this is supposed to be pure removal, I'm uneasy about making other changes beyond the stated purpose of removing PY2 / 3 things.

Reply:
Fine by me. Moved back in the try except.